docs/ja/models/litellm.md

```
@@ -2,17 +2,17 @@
-    LiteLLM 連携はベータ版です。特に小規模なモデルプロバイダーでは問題が発生する場合があります。問題があれば [Github issues](https://github.com/openai/openai-agents-python/issues) にご報告ください。迅速に修正します。
+    LiteLLM 連携はベータ版です。特に規模の小さいモデルプロバイダーでは問題が発生する可能性があります。問題があれば [Github issues](https://github.com/openai/openai-agents-python/issues) からご報告ください。迅速に対応します。
 
-[LiteLLM](https://docs.litellm.ai/docs/) は、単一のインターフェースで 100+ のモデルを利用できるライブラリです。Agents SDK で任意の AI モデルを使えるように、LiteLLM との連携を追加しました。
+[LiteLLM](https://docs.litellm.ai/docs/) は、1 つのインターフェースで 100+ のモデルを利用できるライブラリです。Agents SDK で任意の AI モデルを使えるように、LiteLLM 連携を追加しました。
 
 ## セットアップ
 
-`litellm` が利用可能である必要があります。オプションの `litellm` 依存関係グループをインストールしてください。
+`litellm` が利用可能である必要があります。オプションの `litellm` 依存関係グループをインストールしてください:
 
 ```bash
```

markdownlint-cli2 (0.23.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)